apps/api/src/app/workflows-v2/usecases/create-variables-object/create-variables-object.usecase.ts

```diff
@@ -205,7 +205,7 @@ export class CreateVariablesObject {
     // Extract 'Repeat' block iterable variables ('each' key) together with their set iterations
     const eachKeyVars = this.extractMailyAttribute(controlValues, MailyAttrsEnum.EACH_KEY).map((path) => ({
       path,
-      iterations: this.getIterationsForVariable(path, controlValues) || DEFAULT_ARRAY_ELEMENTS,
```

Comment:
what does this change means?

Reply:
it means that we are not looking for the repeat block `max iterations` but by default generate 5 elements